addons/website/models/website.py

```diff
@@ -1267,3 +1267,77 @@ def _get_cached_values(self):
 
     def _get_cached(self, field):
         return self._get_cached_values()[field]
+
+    def _get_snippets_containers(self):
```

Comment:
```suggestion
    def _get_html_fields(self):
```
What do you think? `_get_snippets_containers` is very confusing, especially since snippets are DOM element, are we talking about div with container class? You have to read the code to figure what container mean

Reply:
I agree, container is not the right term, I wanted to avoid using html fields terminology as we add to the list the arch_db from ir_ui_view which is technically not an orm html field but in the end, all these columns host html elements... changing it !